### ISSUE TITLE
Add icons for person types

### ISSUE THREAD

**Issue by sarahmonster** [a8cteam51/strikebase#107]
Currently I've super-lazily colour-coded people types, but icons would be much better

- [ ] influencer/beneficiary/PDCT
- [ ] striker
- [ ] automattician
- [ ] contractor

**Comment by wrightcj03** [comment]
Looks good!

**Comment by michiecat** [comment]
I vote for "PDCT". :) Otherwise they all look good to me.

**Comment by michiecat** [comment]
Changed "influencer" in to "PDCT" in Strikebase, otherwise the rest were already existing as is in the site.

**Comment by sarahmonster** [comment]
It might make sense to change it back to "influencer" for the time being, since that's hardcoded into the theme in a few places (notably for the front-end filters and also for the fieldmanager fields). We'll need to take this into account (it'd be nice if this weren't hard-coded at all, but I'm not sure the most sensible approach there).

Might be worth an (internal) brainstorming session to finally sort out some terminology that works for us (I'm generally a bit wary of acronyms!)

**Comment by michiecat** [comment]
Yes, I noticed the hardcoding and changed it back. :) I don't have a strong preference for what we call it, so whatever the consensus decides.
